internal/grpctest/tlogger.go

```diff
@@ -41,8 +41,23 @@ const callingFrame = 4
 
 type logType int
 
+func (l logType) String() string {
+	switch l {
+	case infoLog:
+		return "INFO"
+	case warningLog:
+		return "WARN"
+	case errorLog:
+		return "ERRO"
```

Comment:
IMO there's no reason to truncate this or `FATAL`.  Or even `WARNING`.  It's not like we're expecting tons of these lines and hoping they will line up.  It may even be better if they stand out more.

Reply:
Done